### PR TITLE
Eva/lane as obstacle + empty lane detection bug

### PR DESCRIPTION
**Pull request by evamo0508**
fixed two bugs - 
- Lane as obstacles during navigation.
- Lane detection won't stop running even if the center line pointcloud is empty.

### CODE REVIEW

File: autovalet_navigation/config/sim/costmap_common_params.yaml [hunk 1]
```diff
@@ -15,11 +15,25 @@ inflation: # Name of the inflation layer plugin in global costmap params
   cost_scaling_factor: 1.0
 
 obstacle_layer:
+    observation_sources: lane
     max_obstacle_height: 1.5
-    min_obstacle_height: 0.2
-
-observation_sources: realsense velodyne lane
+    min_obstacle_height: -0.2
```
Comment (mishra-nikita) [inline autovalet_navigation/config/sim/costmap_common_params.yaml]: why is min obstacle height negative(-0.2)?

**Review by mishra-nikita — DISMISSED** [review]

File: autovalet_navigation/config/sim/base_local_planner_params.yaml [hunk 2]
```diff
@@ -78,11 +78,11 @@ TebLocalPlannerROS:
  weight_acc_lim_x: 1.0
```
Comment (kob51) [inline autovalet_navigation/config/sim/base_local_planner_params.yaml]: maybe don't push this file? we'll have a better version when i eventually merge my `kevin/planner_2` branch with the best param list

File: autovalet_navigation/config/sim/costmap_common_params.yaml (hunk 1 above)
Comment (evamo0508) [reply autovalet_navigation/config/sim/costmap_common_params.yaml]: you're right. It should be reverted to some positive values. It was changed to do some sanity check. good catch!

File: autovalet_navigation/config/sim/base_local_planner_params.yaml (hunk 2 above)
Comment (evamo0508) [reply autovalet_navigation/config/sim/base_local_planner_params.yaml]: sure I will revert it to its original version

**Review by SubramanianKrish — APPROVED** [review]

**Review by mishra-nikita — APPROVED** [review]

**Pull request merged**